repository/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.eclipse.wildwebdeveloper</groupId>
 		<artifactId>parent</artifactId>
-		<version>0.2.0-SNAPSHOT</version>
+		<version>0.2.1-SNAPSHOT</version>
```

Comment:
Since it's a relatively noticeable addition, we'll bump to 0.3.0 to showcase there is more than maintenance fix.